private/buf/bufgen/internal/bufgenv2/override.go

```diff
@@ -12,35 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package bufimagemodifyv2
+package bufgenv2
 
 import "google.golang.org/protobuf/types/descriptorpb"
 
+type override interface {
+	override()
+}
+
 type prefixOverride string
 
 func newPrefixOverride(prefix string) prefixOverride {
 	return prefixOverride(prefix)
 }
 
-func (p prefixOverride) Get() string {
+func (p prefixOverride) get() string {
```

Comment:
nit: id keep these all as public, since the type is private they are not exposed, and this makes it easy to switch between this type being public and private in the future. I generally use private functions just for truly-internal-to-the-type stuff